docs/notes/2.22.x.md

```diff
@@ -68,6 +68,8 @@ The deprecation for the `platforms` field for the `pex_binary` and `pex_binaries
 
 Python tool subsystem docs and help text now include the default version of the tool, along with instructions on how to override this version using a custom lockfile. Additionally, the help text for the `install_from_resolve` option for Python tools now includes this same information.
 
+Python tools can be exported from their default bundled lockfiles.
```

Comment:
It's minor, but what do you think about explaining this a bit more, not all users may be immediately familiar with the terminology and thus may not realise it's useful for them? For instance:

```suggestion
Python tools can be [exported from their default bundled lockfiles](https://www.pantsbuild.org/2.22/docs/using-pants/setting-up-an-ide#tool-virtualenvs). For instance, when using the default `black` subsystem, `pants export --resolve=black` will export a venv containing the version of black that Pants runs.
```